src/components/cards/RoleCard.vue

```diff
@@ -103,42 +91,43 @@
     })
 </script>
 
-<style scoped>
-  *{    
-    font-family: "Courier New";
-    font-size: 18;
+<style scoped lang="scss">
+  .role--wrapper {
+    font-family: $font--standard;
   }
-  .role-top {
+
+  .role--header {
+    align-items: center;
     background-color: grey;
     border: 2px solid black;
-    font-family: "Courier New";
-    font-size: 18px;
-    padding: 10px;
-    box-sizing: border-box;
-    border-radius:0px!important;
+    display: flex;
```

Comment:
This is actually a bit of a hack, I'm not sure what the hell is going on with the height of this h3 but it's got a huge default size and I was struggling to fix it.  So I gave up and for now, just centered it in a flex layout.